R/env.R

```diff
@@ -778,6 +835,12 @@ env_has <- function(env = caller_env(), nms, inherit = FALSE) {
 env_get <- function(env = caller_env(), nm, inherit = FALSE) {
   get(nm, envir = get_env(env), inherits = inherit)
 }
+#' @rdname env_get
+#' @export
+env_get_list <- function(env = caller_env(), nms, inherit = FALSE) {
```

Comment:
`env_get_many()`?

Reply:
I thought `_list` would be consistent with the return type rule. We already have other `_list` like `dots_list()`.

Reply:
Ok